presses-universitaires-de-paris-nanterre.csl

```diff
@@ -0,0 +1,592 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="note" default-locale="fr-FR" version="1.0" page-range-format="expanded" demote-non-dropping-particle="display-and-sort">
+  <info>
+    <title>Presses universitaires de Paris Nanterre (note, French)</title>
+    <title-short>PU Paris Nanterre</title-short>
+    <id>http://www.zotero.org/styles/presses-universitaires-de-paris-nanterre</id>
+    <link href="http://www.zotero.org/styles/presses-universitaires-de-paris-nanterre" rel="self"/>
+    <link href="http://scd.u-paris10.fr/services/boite-a-outils-pour-la-recherche-/ressources-pratiques-591002.kjsp?RH=1390906967545" rel="documentation"/>
+    <link href="https://presses.u-paris10.fr/wp-content/uploads/2017/02/ConsignesAuteurs.pdf" rel="documentation"/>
```

Comment:
@ajustine, we'd also like to see a "template" link, assuming that you based this CSL style on an existing one. (see https://github.com/citation-style-language/styles/wiki/Style-Requirements#8---template-link)